code/game/objects/items/rogueitems/needle.dm

```diff
@@ -69,7 +69,7 @@
 
 /obj/item/needle/attack_obj(obj/O, mob/living/user)
 	var/obj/item/I = O
-	if(can_repair)
+	if(istype(I) && can_repair)
```

Comment:
Can you leave this one for me pretty please? I'm already retouching this code and don't want to have to fix a merge conflix even if its super easy!

Reference: #1222

Reply:
I also just discovered in our code "can_repair" does absolutely nothing wow

Reply:
AHHH I SEE WHY, The var exists because theres supposed to exist a medic only needle, WHICH MAKES SENSE! I'll probably add that at some point!

Reply:
Will do!